pages/a/upgrade-your-cluster-to-0-55.html

```diff
@@ -0,0 +1,116 @@
+title: Upgrade Your Cluster to 0.55.x
+author: Claus Matzinger, Chris Ward
+description: 0.55 of Crate is a big release and introduces extra steps to
+upgrade.
+created: 2016-07-25
+post_name: upgrade-your-cluster-to-0-55
+status: publish
+post_type: post
+tags: developernews, features
+category: post
+
+Version 0.55 of Crate introduces a lot of great features, fixes and changes.
+Some of these required making changes to our code base that require extra steps
+to upgrade your clusters than with previous Crate versions.
+
+**Note**: Upgrading to 0.55 is only supported from Crate 0.52 and upwards.
+
+## Preparation
+
+Before upgrading a cluster it's strongly advised to [create a
+backup](https://crate.io/a/backing-up-and-restoring-crate/) of your current
+tables so you can always return to what you had before. Furthermore be prepared
+to restart your entire cluster.
+
+## Index Changes
+
+Because of the Elasticsearch version upgrade, indices are not backwards
+compatible to 0.54. Be aware that once upgraded to 0.55, using the same index
+files in a 0.54 cluster will yield an error:
+
+```bash
+[2016-07-22 11:58:12,931][ERROR][gateway.local.state.shards] [Brian Braddock]
+failed to read local state (started shards), exiting...
+org.elasticsearch.ElasticsearchException: unexpected field in shard state
+[index_uuid]
+...
+```
+
+## Settings Changes
+
+There have been several changes and depreciations to settings in 0.55, some due
```

Comment:
Also..is there doc on the changes & deprecations? As a user, how do I learn whether the changes will affect me and what action I should take?